t/xml_report.t

```diff
@@ -308,14 +305,19 @@ my $prove_id = Experian::IDAuth->new(
     phone         => '34878123',
     email         => 'john.galt@gmail.com',
     premise       => 'premise',
+    folder        => $tmp_dir,
 );
 
-warning_like( sub {
-    my $prove_id_result = $prove_id->get_result();
-    my $xml_report = $prove_id->get_192_xml_report();
+warning_like(
+    sub {
+        my $prove_id_result = $prove_id->get_result();
+        my $xml_report      = $prove_id->get_192_xml_report();
+
+        ok($xml_report eq $xml, 'get_192_xml_report');
+    },
+    qr/not a pdf/,
+    'bad pdf warning'
+);
 
```

Comment:
Would be nice to have a test which verifies presence of warning mail in mailbox like:
`
use Email::Folder::Search;
my $mailbox = Email::Folder::Search->new('/tmp/default.mailbox');
$mailbox->init;
my @mails = $mailbox->search(
  email    => 'compliance@binary.com',
  subject => ''Experian request error'
);
ok(@mails, 'Warning sent to compliance team');
like($mails[0]{body}, qr/.*There was an error during Experian request/s, 'Issue is described in email content');
`